### PR TITLE
Scree plot

### PR DESCRIPTION
**Pull request by damirpolat**
I added an option to create scree plots for hierarchical clustering learners. A scree plot can help choose optimal number of clusters.

```
library(mlr3)
library(mlr3cluster)
library(mlr3viz)

task = tsk("usarrests")
learner = lrn("clust.hclust")
learner$train(task)
autoplot(learner, type="scree")
```

scree

### CODE REVIEW

**Review by pat-s — CHANGES_REQUESTED** [review]
Thanks!

Could you please resolve the R CMD Check warnings and also add the {vdiffr} snapshot from the test?

**Comment by damirpolat** [comment]
Should be resolved now! 

Thanks, 
Damir

File: R/LearnerClustHierarchical.R [hunk 1]
```diff
@@ -58,7 +58,8 @@ autoplot.LearnerClustHierarchical = function(object, type="dend", ...) { # nolin
 
 	"scree" = {
 		data = data.table(Height = object$model$height, Clusters = length(object$model$height):1)
-		ggplot(data, aes(x = data$Clusters, y = data$Height)) + geom_point() + geom_line()
+		ggplot(data, aes(x = data$Clusters, y = data$Height)) + geom_point() + geom_line() + 
+			xlab("Clusters") + ylab("Height")
 	})
 }
```
Comment (pat-s) [inline R/LearnerClustHierarchical.R]: Could you please change tabs -> whitespaces here and possibly in other related locations?

Comment (damirpolat) [reply R/LearnerClustHierarchical.R]: Thank you for noticing! I fixed it. It only happened in this file.

**Review by pat-s — APPROVED** [review]
LGTM 🚀️

Do you mind adding an entry to NEWS.md in the main branch describing the additions?

PS: Please make sure to use the `mlr-style` WRT to code styling next time - instructions can be found [here](https://github.com/mlr-org/mlr3/wiki/Style-Guide) :)

**Pull request merged**